docs/idl.md

```diff
@@ -327,3 +327,43 @@ will be `RecordWithConst::CONST_VALUE` in C++, `RecordWithConst.CONST_VALUE` in
 If comments are placed on top or inside a type definition, they will be converted to
 Javadoc / Doxygen compatible comments in the generated Java, C++, Objective-C and C++/CLI interfaces, or a
 Python docstring.
+
+## Deprecation Comments
+
+It's possible to add deprecation comments to IDL types and methods.
+
+The comment in the IDL file has to start the `# @deprecated reason/hing message` .
```

Comment:
Out of curiosity, what's a "hing message"?

Reply:
very probably a hint :-)

Reply:
```suggestion
The comment in the IDL file has to start the `# @deprecated reason/hint message` .
```